released/packages/coq-ltac-iter/coq-ltac-iter.1.1.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "gmalecha@gmail.com"
+homepage: "https://github.com/gmalecha/coq-ltac-iter"
+dev-repo: "git+https://github.com/gmalecha/coq-ltac-iter.git"
+bug-reports: "https://github.com/gmalecha/coq-ltac-iter/issues"
+authors: ["Gregory Malecha"]
+license: "BSD"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["rm" "-R" "%{lib}%/coq/user-contrib/LtacIter"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.8" & < "8.10~"}
+]
+synopsis:
+  "Access hint databases from tactics"
+description: "Provides tactics for performing various forms of iteration over hypotheses, hint databases, and other collections."
+flags: light-uninstall
```

Comment:
and that one cen be removed, since opam 2.0 tracks the installed files and is able to implement simple removal by itself. Please remove the flags: and remove: stanzas.